package.json

```diff
@@ -59,7 +59,7 @@
   },
   "dependencies": {
     "chalk": "2.4.1",
-    "generator-jhipster": "^5.7.1",
+    "generator-jhipster": "5.8.2",
```

Comment:
Can we leave the ^ in so that future versions will be supported. This value is used when checking if the target application is compatible.

Reply:
Done